index.html

```diff
@@ -71,37 +71,37 @@
-            <a onclick="javascript:display_info(2); return false;"><h1 class="desc display-4 text-center text-muted" id="status-text" style="cursor: pointer; border-right: 1px solid gray; padding-left: 2.25vw; padding-right: 2.25vw; font-size: calc(0.4vh + 0.8vw);">Usage</h1></a>
-            <a onclick="javascript:display_info(3); return false;"><h1 class="desc display-4 text-center text-muted" id="status-text" style="cursor: pointer; border-right: 1px solid gray; padding-left: 2.25vw; padding-right: 2.25vw; font-size: calc(0.4vh + 0.8vw);">Credits</h1></a>
-            <a onclick="javascript:display_info(4); return false;"><h1 class="desc display-4 text-center text-muted" id="status-text" style="cursor: pointer; border-right: 1px solid gray; padding-left: 2.25vw; padding-right: 2.25vw; font-size: calc(0.4vh + 0.8vw);">Privacy</h1></a>
-            <a onclick="javascript:display_info(5); return false;"><h1 class="desc display-4 text-center text-muted" id="status-text" style="cursor: pointer;                                 padding-left: 2.25vw; padding-right: 2.25vw; font-size: calc(0.4vh + 0.8vw);">Source</h1></a>
+            <a onclick="javascript:display_info(0); return false;"><h1 class="desc display-4 text-center text-muted">About</h1></a>
+            <a onclick="javascript:display_info(1); return false;"><h1 class="desc display-4 text-center text-muted">Functionality</h1></a>
+            <a onclick="javascript:display_info(2); return false;"><h1 class="desc display-4 text-center text-muted">Usage</h1></a>
+            <a onclick="javascript:display_info(3); return false;"><h1 class="desc display-4 text-center text-muted">Credits</h1></a>
+            <a onclick="javascript:display_info(4); return false;"><h1 class="desc display-4 text-center text-muted">Privacy</h1></a>
+            <a onclick="javascript:display_info(5); return false;"><h1 class="desc display-4 text-center text-muted"style="border-right: 0;">Source</h1></a>
             <div style="margin-left: 10vw"></div>
           </div>
         </div>
       </div>
     </nav>
     <nav class="description-navbar" id="description-navbar" style="overflow: hidden; position: fixed; z-index: 2; background-color: #dfdfdf; height: 0vh; transition: height 0.3s">
-      <p class="display-4 text-center text-muted" id="desctext" style="padding-top: 2vh; padding-left: 20vw; padding-right: 20vw; font-size: calc(0.3vh + 0.78vw)"></p>
+      <p class="display-4 text-center text-muted" id="desctext"></p>
     </nav>
     <div style="padding-bottom: 1.5vh"> </div>
     <div class="row justify-content-center">
-      <div class="row text-center d-sm-flex d-xl-flex justify-content-center" style="text-align: center; align-items: center; width: 100%">
-        <div class="col-md-6" style="width: 25%; max-width: calc(15vw + 20vh); padding-right: 1px">
+      <div class="row text-center d-sm-flex d-xl-flex justify-content-center" id="simview">

```

Comment:
So it hit me that this div (row justify-content-center) is a bit unnecessary since it really only covers one div (#simview).  Could you move #simview up and add a margin-left/right to space it from the sides?